engine/experience.asm

```diff
@@ -6,7 +6,7 @@ CalcLevel: ; 50e1b
 .next_level
 	inc d
 	ld a, d
-	cp (MAX_LEVEL + 1) % $100
+	cp MAX_LEVEL + 1
```

Comment:
This was here so that a max level of 255 would work. Otherwise it's a "not 8 bit" error.

Reply:
Oops, will fix that.